src/super_gradients/training/sg_model/sg_model.py

```diff
@@ -1719,6 +1737,26 @@ def evaluate(self, data_loader: torch.utils.data.DataLoader, metrics: MetricColl
 
         if self.multi_gpu == MultiGPUMode.DISTRIBUTED_DATA_PARALLEL:
             logging_values = reduce_results_tuple_for_ddp(logging_values, next(self.net.parameters()).device)
+
+        logging_values = get_logging_values(loss_avg_meter, metrics, self.criterion)
+        pbar_message_dict = get_train_loop_description_dict(logging_values,
+                                                            metrics,
+                                                            self.loss_logging_items_names)
+
+        for monitored_value_name in self.valid_monitored_values.keys():
```

Comment:
Lets move this to a helper class method.

Reply:
I moved it to a helper function because I had no need for "self" in it, so I thought it would be better like that to avoid overloading the SgModel class